src/openrct2/world/map.h

```diff
@@ -259,6 +259,26 @@ enum
     MAP_ELEM_SMALL_SCENERY_COLOUR_FLAG_NEEDS_SUPPORTS = (1 << 5),
 };
 
+enum {
+    MAP_ELEMENT_SLOPE_NE_CORNER_UP = 1
```

Comment:
I noticed [slope logic](https://github.com/platipo/OpenRCT2/blob/53db03d7e6d8ca718c748428b2c7aec4c38c42fd/src/openrct2/world/map.c#L546) to be defined in the function itself but never declared as enum. Now I was asking myself how to organize such flags, would this be a convenient way?

Yes, I forgot the commas 🤦‍